packages/scripts/src/migration/dvm2-designated-voting/0_Deploy.ts

```diff
@@ -1,111 +1,68 @@
-const hre = require("hardhat");
```

Comment:
I re-worked this file a bunch. mainly addressed some sizing issues and pulled things into a common file.